v03_pipeline/lib/model/dataset_type.py

```diff
@@ -114,6 +145,11 @@ def genotype_entry_annotation_fns(self) -> list[Callable[..., hl.Expression]]:
                 shared.GT,
             ],
             DatasetType.MITO: [
+                mito.contamination,
```

Comment:
`contamination` and `mito_cn` are column fields not entry fields.  Do we need to keep them?  Should we be explicit about which column fields we keep?

Reply:
👍

Reply:
Implemented [here](https://github.com/broadinstitute/seqr-loading-pipelines/pull/543/files#diff-dea15076a8ab788d04e8e4a78eabb479d472e5f389152cea68d0adabe99e141dR69)